src/index.ts

```diff
@@ -141,15 +161,40 @@ async function runCheckOn(
   }
 }
 
+async function runGHACheckOn(context: Context, headSha: string, checkUrl: string, runId: number) {
+  const started_at = new Date();
+  const checkContext: IContext = {
+    bot: context,
+    logger: new Logger(shortid()),
+  };
+  checkContext.logger.info('Starting check run for:', headSha);
+  const check = await createCheck(context, headSha, checkUrl, 'Artifact Comparison');
+  const artifacts = await getGHAArtifacts(checkContext, runId);
+  await updateCheckFromArtifacts(context, artifacts, started_at, check.data.id, checkContext);
+}
+
 const probotRunner: ApplicationFunction = (app) => {
   app.on(
-    ['pull_request.opened', 'pull_request.reopened', 'pull_request.synchronize'],
+    [
+      'check_run.completed',
+      'pull_request.opened',
+      'pull_request.reopened',
+      'pull_request.synchronize',
+    ],
     async (context) => {
-      const headSha = context.payload.pull_request.head.sha;
-      const baseBranch = context.payload.pull_request.base.ref;
-      const forkRemote = context.payload.pull_request.head.repo.clone_url;
-
-      runCheckOn(context, headSha, baseBranch, forkRemote);
+      if (context.name === 'pull_request') {
```

Comment:
my understanding is `name` is one of valid `WebHookEvents` - 
<img width="854" alt="Screenshot 2024-09-16 at 3 50 24 PM" src="https://github.com/user-attachments/assets/def13a42-2379-4e48-b315-b65a399fb01a">

would this be `context.name.startsWith` maybe?